src/test/shamap/SHAMapSync_test.cpp

```diff
@@ -194,7 +194,7 @@ class sync_test : public beast::unit_test::suite
 
         BEAST_EXPECT(source.deepCompare (destination));
 
-        std::cerr << "Checking destination invariants" << std::endl;
+        log << "Checking destination invariants..." << std::endl;
```

Comment:
I wonder if this output could be removed altogether.  Is it only there because `invariants()` will `assert` so this provides some guide to the call site?